src/main/java/seedu/address/model/commons/Name.java

```diff
@@ -10,13 +10,14 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain alphanumeric characters and spaces and [s/o][d/o](case insensitive),"
```

Comment:
Same thing here, we can be more specific for the constraints of s/o and d/o. Can follow the email message constraint format.